EarlGrey/EarlGrey.h

```diff
@@ -69,6 +69,7 @@
 #import <EarlGrey/GREYTestHelper.h>
 #import <EarlGrey/GREYUIThreadExecutor.h>
 #import <Foundation/Foundation.h>
+#import <EarlGrey/GREYMultiFingerSwipeAction.h>
```

Comment:
nit: Imports here are sorted. Could you add this new import to it's right place?